lib/balance-tx/test/spec/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -1206,15 +1210,21 @@ prop_balanceTxValid
                     ">20 payment outputs"
                 . classify (length originalOuts > 100)
                     ">100 payment outputs"
-
+        let coverage res =
+                cover 10 (isRight res) "success"
```

Comment:
After some experimentation, I found that reducing the coverage to `8%` generally requires between 800 and 1600 test runs:

```suggestion
        let coverage res =
                checkCoverage .
                cover 8 (isRight res) "success"
```